B827EBFFFE26CBE9.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFE26CBE9", 
+		"servers": [ 
+			{ 
+				"server_address": "router.eu.thethings.network", 
+				"serv_port_up": 1700, 
+				"serv_port_down": 1700, 
+				"serv_enabled": true 
+			} 
+		], 
+		"ref_latitude": 57,76628134, 
```

Comment:
Decimal numbers should use "." not ",". This won't work. Please update it and we can merge afterwards. :)